tools/plugin/alsaplug/tplg.c

```diff
@@ -270,7 +270,7 @@ static int plug_new_pga(snd_sof_plug_t *plug)
 
 	/* skip kcontrols for now */
 	if (tplg_create_controls(ctx, ctx->widget->num_kcontrols,
-				 tplg_ctl, ctx->hdr->payload_size, &volume) < 0) {
+				 tplg_ctl, ctx->hdr->payload_size, comp_info) < 0) {
```

Comment:
usually when a parameter is changed the function should be changed too, unless the parameter is ignored there

Reply:
@lyakh this parameter has been unused so far. It will only be used in the following patch